packages/o/openvpn3/monitoring.yaml

```diff
@@ -0,0 +1,7 @@
+---
+releases:
+  id: 378426
+
```

Comment:
Codeberg/Forgejo supports RSS for releases and tags: https://codeberg.org/OpenVPN/openvpn3-linux/tags.rss